web-client/public/js/graph.js

```diff
@@ -57,9 +52,12 @@ export var drawGraph = function (network, sliderController, nodeColoring) {
 
     var minimumScale = MIN_SCALE;
 
+    /* If grnState.colorOptimal is false, then weighting is ignored,
+    and the lines are all drawn as if it was an unweighted sheet */
+    // TODO: incorporate into grnState
     var allWeights = network.positiveWeights.concat(network.negativeWeights);
 
-    if (!colorOptimal) {
+    if (!grnState.colorOptimal) {
```

Comment:
_This_ change I was envisioning for the _next_ iteration, not this one. Remember, _we are limiting the size of our units of work_. Looks like we have some further calibration to do in terms of determining the right boundaries for that. FWIW I viewed the `TODO` comment as something for _this_ PR so that we don't forget; but the actual work of _moving_ `colorOptimal` would be a future round.

Reply:
Remember, we don't want to "pollute" our old code with new code; we want, as much as possible, to just be moving old code _out_ of there and into the new framework. So, rather than introducing `grnState` here, it's cleaner to move `colorOptimal` _out_ completely in a future round of work. (future as in, within the next few PRs)